.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,15 +22,14 @@
 ## Checklist
 <!--- Go over all the following points, and put an `x` in all the boxes that apply -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] I have read the **CONTRIBUTING** document
+- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document
```

Comment:
Does the relative URL (`../`) work when the template is copied to PR description?

Reply:
Yeah, it should work unless contributing file is deleted from their forked version, but I'll change it to the absolute link so that it always links to the latest contributing file.